website/docs/components/breadcrumb/partials/code/component-api.md

```diff
@@ -1,56 +1,52 @@
-The `Breadcrumb` component is composed by different parts, with their own APIs:
+## Component API
 
-*   a main "container" (the breadcrumb itself)
-*   an "item" sub-component (a single "crumb")
-*   a "truncation" sub-component (a hidden list of "crumbs" that can be made visible via a toggle)
+The breadcrumb component is composed of three different parts, each with their own APIs:
 
-#### Breadcrumb
+- a main "container" (the breadcrumb itself)
+- an "item" sub-component (a single "crumb")
+- a "truncation" sub-component (a hidden list of "crumbs" that can be made visible via a toggle)
 
-Here is the API for the main ("container") component:
+### Breadcrumb container
 
 <Doc::ComponentApi as |C|>
-  <C.Property @name="itemsCanWrap" @type="boolean">
-    This controls if the breadcrumb items can wrap in case they can't fit in the container width.
-  </C.Property>
-  <C.Property @name="didInsert" @type="function">
-    This hook method is called when the component is inserted in the DOM. _Notice: internally we use the "did-insert" modifier from @ember/render-modifiers._
-  </C.Property>
-  <C.Property @name="...attributes">
-    `...attributes` spreading is supported on this component. _Notice: by default an attribute `aria-label="breadcrumbs"` is assigned to the component. If you want to localize it you can override it passing the same attribute with a different value._
-  </C.Property>
+<C.Property @name="itemsCanWrap" @type="boolean">
+This controls if the breadcrumb items can wrap in case they don't fit in the container width.
+</C.Property>
+<C.Property @name="didInsert" @type="function">
+This hook method is called when the component is inserted in the DOM. _Note: internally we use the "did-insert" modifier from `@ember/render-modifiers`._
```

Comment:
```suggestion
This hook method is called when the component is inserted in the DOM. 

!!! Info

**Note**

Internally, we use the `did-insert` modifier from `@ember/render-modifiers`.

!!!
```

Reply:
@MelSumner I did try this initially, but it doesn't seem like a markdown invocation of the banner works inside of the `ComponentApi` component. Unless you got it working, then I'd love to pair with you on it!